### PR TITLE
Add full hash example

### PR DESCRIPTION
**Pull request by mcbrewster**
Screen Shot 2019-06-04 at 2 06 05 PM

The new example looks identical to the old one however no calls to app.js are made instead, instead the full hash of the essence is appended to the URL to update the link.

### CODE REVIEW

File: control-with-full-hash/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+In this example the user can search in either datacube or dashboard view to update an iframe with a new Imply-UI URL that applies a page filter. In datacube view the user also has the option of selecting a dimension from a dropdown menu to generate new URL and update the iframe. 
+
+![screenshot of control-with-url-example](images/data_cube.png "control-with-url-example")
+![screenshot of control-with-url-example](images/dashboard.png "control-with-url-example")
+
+To run: 
+
+`npm install`
+
+`npm start`
```
Comment (vogievetsky) [inline control-with-full-hash/README.md]: Why do we need npm install for this example?

File: control-with-full-hash/public/index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<!doctype html>
+<html lang="en">
+    <head>
```
Comment (vogievetsky) [inline control-with-full-hash/public/index.html]: could you dedent this by one?

File: control-with-full-hash/public/logic.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  splits = [];
+
+  if (show) {
+    view = 'dataCube';
+
+    // Set DataCube default Url
+    defaulturl = 'http://localhost:9095/pivot/d/druid_wikipedia';
+
+    path = '/mkurl-dataCube'
+    document.getElementById('dropDown').style.display = 'block';
```
Comment (vogievetsky) [inline control-with-full-hash/public/logic.js]: please have all ids be kebab case. I.e. `drop-down`

File: control-with-full-hash/public/logic.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+function setDimension(type){
+  splits = [{
+    dimension: (type.toLocaleLowerCase()),
+    sortType: "measure",
+    direction: "descending"
+  }];
+
+  // updated displayed dimension to selected dimension
+  document.getElementById('dropdownMenuButton').textContent=type;
```
Comment (vogievetsky) [inline control-with-full-hash/public/logic.js]: kebab please

File: control-with-full-hash/README.md (hunk 1 above)
Comment (mcbrewster) [reply control-with-full-hash/README.md]: I left it because of express. Should we not be using express entirely anymore ?

**Comment by vogievetsky** [comment]
yeah use `http-server` https://www.npmjs.com/package/http-server

**Comment by vogievetsky** [comment]
gr 🎱

**Pull request merged**